platform/keyword-registry/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+  <bean id="keywordRegistry" class="org.codice.ddf.keyword.KeywordRegistryImpl">
+    <cm:managed-properties
+      persistent-id="org.codice.ddf.keyword.KeywordRegistryImpl"
+      update-strategy="container-managed"/>
+
+    <property name="keywordList">
+      <list>
+        <value>Data Source=Data Source</value>
+        <value>Source=Source</value>
+        <value>source=source</value>
+        <value>Sources=Sources</value>
+        <value>sources=sources</value>
```

Comment:
Why are you setting these as default when they don't do anything?

Reply:
By removing the keyword “Source” from the application and by setting default configurable values we decouple the system from the keyword. Otherwise all over the code base we’d see things like

`properties.keywords[‘Source’] || ‘Source’`